api/src/controllers/utils.ts

```diff
@@ -134,43 +131,29 @@ export const getTimespanRange = (
 
       const isNowWeekend = now > friday5PM;
 
-      // This is wrong
-      // An event is this weekend if it occurs during the weekend, not if it starts from Friday 5pm
-      // e.g. an event that lasts 3 days and started Thursday should be included in this.
-      // Relation needed?
-      // Need to test an event that starts Thursday and ends Saturday and if queried on the Monday prior shows up as "this weekend"
-      //
-      // End date: greater than now
       return [
         {
           range: {
             'filter.times.startDateTime': {
-              gte: isNowWeekend ? 'now' : friday5PM, // Friday 5pm or NOW
               lte: now.startOf('week').plus({ days: 6 }).endOf('day'), // Sunday
-              relation: 'contains',
             },
           },
         },
         {
           range: {
             'filter.times.endDateTime': {
-              gt: 'now',
+              gt: isNowWeekend ? 'now' : friday5PM, // Friday 5pm or NOW
```

Comment:
nit: I'd say this comment isn't necessary because the variables are descriptive enough